### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by GlendaChong**
- Add use cases

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -502,148 +502,174 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 
-    Use case ends.
 
--   3a. The given order (ascending or descending) is invalid or not specified.
+**Use Case: UC13 - View Help**
 
-    -   3a1. AddressBook shows an error message.
+**MSS** <br>
+1. User requests for help to use LinkMeIn. 
+2. LinkMeIn displays a link to the user guide. <br>
```
Comment (tayruxin) [inline docs/DeveloperGuide.md:668]: Should be "Displays a window with a link to the user guide"? Will that make more sense

**Review by tayruxin — APPROVED** [review]
Just a really small thing

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -502,148 +502,174 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+**MSS** <br>
+1. User requests to list all companies. 
+2. LinkMeIn displays the full list of companies. <br>
+Use case ends.
 
-**Extensions**
+**Extensions** <br>
+* 1a. LinkMeIn detects an invalid command format error in the input.
+  * 1a1. LinkMeIn displays an error message. 
+  * 1a2. User enters a new command. <br>
```
Comment (alientian) [inline docs/DeveloperGuide.md:521]:
```suggestion
  * 1a2. User enters a new command to list all companies. <br>
```
Not sure if its ok to be vague. If the user enters some other command then step 2 does not make sense

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -502,148 +502,174 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+  Use case resumes from Step 1.
```
Comment (alientian) [inline docs/DeveloperGuide.md:522]: Should be resume from step 2? Since step 1a2 the user already entered a command.
Same for the other use cases too

**Review by alientian — COMMENTED** [review]
Need to change the step which the use case resumes from

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (GlendaChong) [reply docs/DeveloperGuide.md:519]: For this, but if we go to step 2, then it's assuming that the new command is correct? which might not be the case?

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (GlendaChong) [reply docs/DeveloperGuide.md:518]: okie will edit this!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (GlendaChong) [reply docs/DeveloperGuide.md:665]: yup okay!

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (tayruxin) [reply docs/DeveloperGuide.md:519]: how about "Steps 1a1-1a2 are repeated until user enters a valid command format" "Use case resumes from step 2"

**Pull request merged**